jupyterhub/app.py

```diff
@@ -1095,7 +1095,9 @@ def user_stopped(user):
                 # if user.server is defined.
                 log = self.log.warning if user.server else self.log.debug
                 log("%s not running.", user.name)
-                user.server = None
+                for server in user.servers:
```

Comment:
It may make sense to clarify that all servers will be removed when users have more than one server.